modules/overriding-default-node-ip-selection-logic.adoc

```diff
@@ -0,0 +1,77 @@
+// This is included in the following assemblies:
+//
+// ipi-install-configuration-files.adoc
+
+:_content-type: PROCEDURE
+[id="proc-overriding-default-node-ip-selection-logic_{context}"]
+= Optional: Overriding the default node IP selection logic 
+
+You can create a hint file that includes the `NODEIP_HINT` variable to override the default IP selection logic. This allows users to select a specific node IP address from the subnet `NODEIP_HINT` variable and specify which IP address is used. For example, if a node has two interfaces, `eth0` and `eth1`, with the IP addresses `10.0.0.10/24` and `192.0.2.5/25` respectively, and the default route points to `eth0`, the node IP address normally uses the `10.0.0.10` address. With `NODEIP_HINT`, you can specify that the node IP address use `192.0.2.5` by configuring `NODEIP_HINT` to point to `192.0.2.1`. 
+
+Use the following procedure to override the default node IP selection logic. 
+
+.Procedure
+
+. Add a hint file to your your `/etc/default/nodeip-configuration` file. The contents of your hint file must match the following example: 
++
+[source,text]
+----
+NODEIP_HINT=<node_ip_address>
+----
++
+[IMPORTANT]
+====
+* Do not use the exact IP address of a node as a hint, because `runtimecfg` treats it as a VIP and ignores it when looking for a node IP address. This causes the node using the hint IP address to fail to configure correctly. For example, the desired node IP address is `192.0.2.5`, use `192.0.2.1`. 
```

Comment:
What's `runtimecfg`? Looks like a field, but a field doesn't exercise agency. What treats the hint as a VIP?